src/poprox_recommender/data/mind.py

```diff
@@ -94,8 +94,8 @@ def split_records():
             split_records(),
             columns=["mind_item_id", "rating"],
         )
-        truth["item"] = [self.news_uuid_for_id(aid) for aid in truth["mind_item_id"]]
-        return truth.set_index("item")
+        truth["item_id"] = [self.news_uuid_for_id(aid) for aid in truth["mind_item_id"]]
+        return truth.set_index("item_id")
 
```

Comment:
I think we need to make the corresponding change in the POPROX loader as well.

Reply:
I made the change.